[SQL][HIVE] Correct an assert message in function makeRDDForTable

SongYadong opened the pull request:
## What changes were proposed in this pull request?
according to the context, "makeRDDForTablePartitions" in assert message should be "makeRDDForPartitionedTable", because "makeRDDForTablePartitions" does't exist in spark code.


## How was this patch tested?
unit tests


Please review http://spark.apache.org/contributing.html before opening a pull request.

SongYadong committed:
correct an assert info in function makeRDDForTable

according to the context, "makeRDDForTablePartitions" in assert message should be "makeRDDForPartitionedTable"

AmplabJenkins commented:
Can one of the admins verify this patch?

AmplabJenkins commented:
Can one of the admins verify this patch?

AmplabJenkins commented:
Can one of the admins verify this patch?

HyukjinKwon reviewed (commented):
LGTM

HyukjinKwon commented:
ok to test

HyukjinKwon commented:
test this please

HyukjinKwon commented:
retest this please

SparkQA commented:
**[Test build #93427 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/93427/testReport)** for PR 21836 at commit [`6a0f729`](https://github.com/apache/spark/commit/6a0f7290a7ba2764ab9dd437a07afdf419ce7bf0).

SparkQA commented:
**[Test build #93427 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/93427/testReport)** for PR 21836 at commit [`6a0f729`](https://github.com/apache/spark/commit/6a0f7290a7ba2764ab9dd437a07afdf419ce7bf0).
 * This patch **fails Scala style tests**.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Merged build finished. Test FAILed.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/93427/
Test FAILed.

SongYadong committed:
Solve Scala style checks failure.

the line modified exceeds 100 characters, it leads to Scala style checks failure, cut it into two line to solve it.

HyukjinKwon commented on sql/hive/src/main/scala/org/apache/spark/sql/hive/TableReader.scala:
```diff
@@ -111,7 +111,8 @@ class HadoopTableReader(
 
     assert(!hiveTable.isPartitioned, """makeRDDForTable() cannot be called on a partitioned table,
-      since input formats may differ across partitions. Use makeRDDForTablePartitions() instead.""")
+      since input formats may differ across partitions. Use makeRDDForPartitionedTable() 
```
I think it's gonna keep the white spaces as is. Mind if I ask change this as below?

```
assert(
  !hiveTable.isPartitioned,
  "makeRDDForTable() cannot be called on a partitioned table, since input formats may differ " +
    "across partitions. Use makeRDDForTablePartitions() instead."
```

SongYadong replied:
That's Good. I will modify it, thanks.

SongYadong committed:
Fix scala style checks failure in another way.

HyukjinKwon commented:
test this please

SparkQA commented:
**[Test build #93432 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/93432/testReport)** for PR 21836 at commit [`085ffdb`](https://github.com/apache/spark/commit/085ffdb9d87bc698e8f481dd39c5c72c596d3307).

SparkQA commented:
**[Test build #93432 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/93432/testReport)** for PR 21836 at commit [`085ffdb`](https://github.com/apache/spark/commit/085ffdb9d87bc698e8f481dd39c5c72c596d3307).
 * This patch **fails Scala style tests**.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Merged build finished. Test FAILed.

AmplabJenkins commented:
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/93432/
Test FAILed.

SongYadong committed:
Delete whitespace at end of line

SongYadong commented:
I'm sorry for the scala style checks failure, i will read scala style guide doc again.

HyukjinKwon commented:
retest this please

SparkQA commented:
**[Test build #93434 has started](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/93434/testReport)** for PR 21836 at commit [`7dc889b`](https://github.com/apache/spark/commit/7dc889b4d3f9ee9323a969d1ea98f5614e8a27d3).

SparkQA commented:
**[Test build #93434 has finished](https://amplab.cs.berkeley.edu/jenkins/job/SparkPullRequestBuilder/93434/testReport)** for PR 21836 at commit [`7dc889b`](https://github.com/apache/spark/commit/7dc889b4d3f9ee9323a969d1ea98f5614e8a27d3).
 * This patch passes all tests.
 * This patch merges cleanly.
 * This patch adds no public classes.

AmplabJenkins commented:
Merged build finished. Test PASSed.

AmplabJenkins commented:
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
https://amplab.cs.berkeley.edu/jenkins//job/SparkPullRequestBuilder/93434/
Test PASSed.

HyukjinKwon commented:
Merged to master and branch-2.3.
